client/my-sites/guided-transfer/host-credentials-page/bluehost.jsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import {
+	Username,
+	Password,
+	CreateAccountTip,
+	SubmitSection,
+	DestinationURL,
+} from './fields';
+
+class Bluehost extends Component {
+	render() {
+		const {
+			hostInfo,
+			translate,
+			fieldValues,
+			onFieldChange,
+		} = this.props;
+
+		return (
+			<div>
+				<CompactCard>
+					<p>
+						{ translate(
+							'Please enter your credentials. They will be stored securely so that one ' +
+							'of our Happiness Engineers can get the transfer going for you.' ) }
+					</p>
+					<div>
+						<Username
+							value={ fieldValues.username }
+							onChange={ onFieldChange( 'username' ) }
+							hostLabel={ hostInfo.label }
+						/>
+						<Password
+							value={ fieldValues.password }
+							onChange={ onFieldChange( 'password' ) }
+							hostLabel={ hostInfo.label }
+						/>
+					</div>
+					<CreateAccountTip
+						hostLabel={ hostInfo.label }
+						hostUrl={ hostInfo.url } />
+					<DestinationURL
+						value={ fieldValues.wporg_url }
+						onChange={ onFieldChange( 'wporg_url' ) }
+					/>
+				</CompactCard>
+				<SubmitSection submit={ this.props.submit } />
+			</div>
+		);
+	}
+}
+
+Bluehost.propTypes = {
```

Comment:
Good point, now that we have ES7+ property initializers, I've moved back it into the class in 84dd99046f7bb892ee896e17a8a1c9f0df80f40b.

Previously, setting it outside the class was the only way to set the PropTypes on the class [when using ES6 style classes](https://facebook.github.io/react/docs/reusable-components.html#es6-classes) instead of `React.createClass`.